printing/printer.ml

```diff
@@ -471,7 +471,8 @@ let pr_goal ?(diffs=false) ?og_s g_s =
   let env, concl = goal_repr sigma g in
   let goal =
     if diffs then
-      Proof_diffs.diff_goal ?og_s g sigma
+      let g = Proof_diffs.make_goal (Global.env ()) sigma g in
```

Comment:
What's the difference between env and Global.env here?

Reply:
It does not matter for this function call, but `env` is Global.env where the named context have been set to the one of the evar.